cmd/util/cmd/execution-state-extract/execution_state_extract_test.go

```diff
@@ -67,7 +67,7 @@ func TestExtractExecutionState(t *testing.T) {
 			blockID := unittest.IdentifierFixture()
 			stateCommitment := unittest.StateCommitmentFixture()
 
-			err := unittest.WithLock(t, lockManager, storage.LockInsertOwnReceipt, func(lctx lockctx.Context) error {
+			err := unittest.WithLock(t, lockManager, storage.LockIndexStateCommitment, func(lctx lockctx.Context) error {
```

Comment:
Instead reuse the same lock, I think we should acquire lock that is specific to the data we are updating.